backend/danswer/chat/answer.py

```diff
@@ -206,7 +206,9 @@ def _get_response(self, llm_calls: list[LLMCall]) -> AnswerStream:
         # + figure out what the next LLM call should be
         tool_call_handler = ToolResponseHandler(current_llm_call.tools)
 
-        search_result = SearchTool.get_search_result(current_llm_call) or []
+        search_result, displayed_search_results_map = SearchTool.get_search_result(
+            current_llm_call
+        ) or ([], {})
```

Comment:
Pref ([], []) for consistency and building the ordering in the call below similar to the other object. For reducing complexity for future readers